ksql-serde/src/main/java/io/confluent/ksql/serde/SerdeOption.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.serde;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+public enum SerdeOption {
+
+  /**
+   * If the value schema contains only a single field, persist it as an anonymous value.
+   *
+   * <p>If not set, any single field value schema will be persisted within an outer object, e.g.
+   * JSON Object or Avro Record.
+   */
+  UNWRAP_SINGLE_VALUES;
```

Comment:
`EnumMap` is mutable, so not something I'm keen to use, plus I don't think we're particularly worried about the space savings the map type offers.

We could use an `ImmutableMap<SerdeOption, boolean>` but this just feels weird / like overkill.  Sure, this is the opposite of the syntax, but I don't think it's causing that much brain tax. 

How strongly do you feel about this?

Reply:
I agree the map is funky. It's just a little annoying to keep switching the way I think about it (especially because I always get confused with the original config as well because I usually think things default to false, so when we have `WRAP_SINGLE_FIELDS=true` I think that it's something different than the default, but it's actually the default...)